jane/doc/extensions/_04-modes/intro.md

```diff
@@ -183,5 +183,64 @@ they are once.
 More documentation on uniqueness and linearity is available
 [here](../../uniqueness/intro/).
 
-<!-- CR ccasinghino: Sections needed for statefulness, visibility, and yielding -->
+## Future modes: Yielding
 
+|----------------|
+| yielding       |
+| `|`            |
+| **unyielding** |
+{: .table }
+
+Yielding is a future axis that tracks whether a function is permitted to perform
+effects that will be handled in its parent stack. See [the OCaml Manual entry
+for effect handlers](https://ocaml.org/manual/5.3/effects.html).
+
+Yielding has different defaults depending on the locality axis: *global* values are
+defaulted to *unyielding*, while *local* values are defaulted to *yielding*.
+More documentation on mode implications is available [here](../_05-kinds/syntax.md).
```

Comment:
Is that the right link, with the underscore?

And is that the only place the documentation on implications is written? It also applies to mode and modality expressions, right? Not just kinds. Seems like a problem somewhere.

Reply:
Yeah, that's the only place. The directory names include underscores.

Reply:
<<resolve>>